snippets/logging-alertingrule-app-callouts-snip.adoc

```diff
@@ -0,0 +1,31 @@
+.Example application AlertingRule CR
+[source,yaml]
+----
+  apiVersion: loki.grafana.com/v1
+  kind: AlertingRule
+  metadata:
+    name: app-user-workload
+    namespace: app-ns <1>
+    labels: <2>
+      openshift.io/cluster-monitoring: "true"
+  spec:
+    tenantID: "application"
+    groups:
+      - name: AppUserWorkloadHighError
+        rules:
+          - alert:
+            expr: | <3>
+            sum(rate({kubernetes_namespace_name="app-ns", kubernetes_pod_name=~"podName.*"} |= "error" [1m])) by (job)
+            for: 10s
+            labels:
+              severity: critical <4>
+            annotations:
+              summary:  <5>
+              description:  <6>
+----
+<1> The `namespace` where this AlertingRule is created needs to have a label matching the LokiStack `spec.rules.namespaceSelector` definition.
+<2> The labels block needs to match the LokiStack `spec.rules.selector` definition.
```

Comment:
```suggestion
<1> The `namespace` where this AlertingRule is created must have a label matching the LokiStack `spec.rules.namespaceSelector` definition.
<2> The `labels` block must match the LokiStack `spec.rules.selector` definition.
```

Reply:
You can ignore suggestions to replace "needs to" with "must." These suggestions are purely for clarity and minimalism - not an error.